addons/project/controllers/portal.py

```diff
@@ -63,7 +63,7 @@ def _project_get_page_view_values(self, project, access_token, page=1, date_begi
         # task count
         task_count = Task.search_count(domain)
         # pager
-        url = "/my/project/%s" % project.id
+        url = "/my/projects/%s" % project.id
```

Comment:
I think we should keep the current routes and call the new ones from there. We could then remove the old ones when it seems reasonable that no mail message will remain used with the old refs.